test/verify/check-networkmanager-firewall

```diff
@@ -127,6 +127,32 @@ class TestFirewall(NetworkCase):
             ".*The name org.fedoraproject.FirewallD1 was not provided by any .service files.*",
             ".*org.fedoraproject.FirewallD1: .*: GDBus.Error:org.freedesktop.DBus.Error.NoReply.*")
 
+    # test missing "pkcheck" binary, in that case fallback to the admin check
+    def testPkcheckMissing(self):
+        b = self.browser
+        m = self.machine
+
+        # Path to the "pkcheck" PolicyKit utility
+        pkcheck = m.execute("which pkcheck").rstrip("\n")
+
+        if pkcheck:
```

Comment:
Let's drop this. First, it's a no-op -- `which` will fail if pkcheck is not installed. Second, `if` in tests is a code smell in general, tests should be predictable. I'll try a round of tests without the `if`, and see where it's missing (possibly on Fedora CoreOS?)